code-challenges.test.js

```diff
@@ -16,52 +16,232 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
 // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
+// --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.  
 
 // Pseudo code:
+// input: takes in an OBJECT as an ARGUMENT -> function ( object) => {}
+// output: values evenly divisible by 3 or not
+// HOW? if... num % 3 === 0 (remainder) -> divisible , else if...num % 3 === 1 (remainder) -> not
+
+
+
+      
+
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const object1 = { number: 15 }
-// Expected output: "15 is divisible by three"
-const object2 = { number: 0 }
-// Expected output: "0 is divisible by three"
-const object3 = { number: -7 }
-// Expected output: "-7 is not divisible by three"
+
+// describe(`divisibleBy`,  () => {
+//     it(`should return a string indicating if the number is divisible by three or not`, () => {
+//         const object1 = { number: 15 }
+//         const object2 = { number: 0 }
+//         const object3 = { number: -7 }
+//       expect(divisibleBy(object1)).toEqual("15 is divisible by three");
+//       expect(divisibleBy(object2)).toEqual("0 is divisible by three");
+//       expect(divisibleBy(object3)).toEqual("-7 is not divisible by three");
+//     })
+//   })
+
+//   FAIL  ./code-challenges.test.js
+//   divisibleBy
+//     ✕ should return a string indicating if the number is divisible by three or not (1 ms)
+
+//   ● divisibleBy › should return a string indicating if the number is divisible by three or not
+
+//     ReferenceError: divisibleBy is not defined
+
 
 // b) Create the function that makes the test pass.
 
+
+describe(`divisibleBy`,  () => {
+            it(`should return a string indicating if the number is divisible by three or not`, () => {
+              expect(divisibleBy(object1)).toEqual("15 is divisible by three");
+              expect(divisibleBy(object2)).toEqual("0 is divisible by three");
+              expect(divisibleBy(object3)).toEqual("-7 is not divisible by three");
+            })
+          })
+          
+        class Test {                         // Had issues returnig the second output when using the if, else statements by themself; so I created a class to wrap the information. 
```

Comment:
There is no need to create this class at all, as it is basically the same thing as the given variables. You are just giving the object another key value pair of `answer`, which is making an evaluation of if the number is divisible by 3, but that could have just gone in the conditional.